Homework2/build/src/homework2/EditFood.java

```diff
@@ -0,0 +1,68 @@
+package homework2;
+
+import java.util.List;
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+
+
+import javax.servlet.annotation.WebServlet;
+
+@WebServlet("/admin/food/edit")
+public class EditFood extends HttpServlet {
+	public void doGet( HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+			int id = Integer.parseInt(request.getParameter("id"));
+			List<CreateFood> entries = (List<CreateFood>) getServletContext().getAttribute("entries");
+			CreateFood leEntry = null;
+			for (CreateFood entry: entries) {
+				if (entry.getId() == id) {
+					leEntry = entry;
+				}
+			}
+		
+   
+
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		out.println("<body style='background-color:#E6E6FA;'>");
+		out.println("<h1>Tom's #1 World Famous Chiliburgers</h1>");
+		out.println(
+				"<img  src=http://nebula.wsimg.com/38c7cffac476fc379ec83d5ec7f94485?AccessKeyId=ED591A06CCEC51484249&disposition=0&alloworigin=1 alt= logo /> </br>");
+		out.println("<form method=\"post\"></br>");
+		out.println("Food name: <p><input name='name' type='text' value='" + leEntry.getName() + "'/></br></p>");
+		out.println("Food price:<p> <input price='price' type='text' value='" + leEntry.getPrice() + "'/></br><p>");
```

Comment:
The functionality to edit food is probably not completed since the name attribute is not given here for the food price.